reason-react-native/src/apis/Animated.re

```diff
@@ -239,10 +239,26 @@ external loop: Animation.t => Animation.t = "";
 [@bs.module "react-native"] [@bs.scope "Animated"]
 external loopWithConfig: (Animation.t, loopConfig) => Animation.t = "loop";
 
-type animatedEvent;
+type eventOptions;
+[@bs.obj]
+external eventOptions:
+  (~listener: 'event => unit=?, ~useNativeDriver: bool=?, unit) => eventOptions =
+  "";
+
+type eventOptions2;
+[@bs.obj]
+external eventOptions2:
+  (~listener: ('event, 'option2) => unit=?, ~useNativeDriver: bool=?, unit) =>
+  eventOptions2 =
+  "";
+
+type event('event) = 'event => unit;
+
+[@bs.module "react-native"] [@bs.scope "Animated"]
+external event: (array('mapping), eventOptions) => event('event) = "";
```

Comment:
To match `useCallbackX`, `useEffectX`etc., maybe this should be called `event1`?

Reply:
Yeah I was thinking about that. Will make the change.